components/style/custom_properties.rs

```diff
@@ -697,7 +697,7 @@ fn substitute_all(custom_properties_map: &mut CustomPropertiesMap) {
         // Add new entry to the information table.
         let index = context.count;
         context.count += 1;
-        debug_assert!(index == context.var_info.len());
+        debug_assert_eq!(index, context.var_info.len());
```

Comment:
@emilio I just `rg` for `debug_assert!` with `==`. Maybe we can have an `E-easy` issue for it!

```sh
→ rg debug_assert! | rg == | wc -l
      67
```

And for not equal

```sh
→ rg debug_assert! | rg != | wc -l
      21
```

Reply:
Cool

Reply:
Sounds good!